<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MWLTPF1MEAFD" failures="0" tests="5" name="automationassignment.TestCase2Bonus" time="6.179" errors="1" timestamp="2019-10-06T11:17:24 SAST" skipped="4">
  <testcase classname="automationassignment.TestCase2Bonus" name="step3_AddItemsToBasket" time="0.000">
    <skipped/>
  </testcase> <!-- step3_AddItemsToBasket -->
  <testcase classname="automationassignment.TestCase2Bonus" name="step2_SearchForRestaurantsNearAddress" time="0.000">
    <skipped/>
  </testcase> <!-- step2_SearchForRestaurantsNearAddress -->
  <testcase classname="automationassignment.TestCase2Bonus" name="step1_EnterAddress" time="6.179">
    <error message="String index out of range: -11" type="java.lang.StringIndexOutOfBoundsException">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: String index out of range: -11
at java.base/java.lang.String.substring(String.java:1850)
at utility.GeneralSelects.selectItemFromWebElementlist(GeneralSelects.java:26)
at pages.PageTimeToOrderFood.selectSuggestedAddressFromList(PageTimeToOrderFood.java:43)
at pages.PageTimeToOrderFood.enterAddress(PageTimeToOrderFood.java:33)
at automationassignment.TestCase2Bonus.step1_EnterAddress(TestCase2Bonus.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
]]>
    </error>
  </testcase> <!-- step1_EnterAddress -->
  <testcase classname="automationassignment.TestCase2Bonus" name="step5_ReceiveOrderReference" time="0.000">
    <skipped/>
  </testcase> <!-- step5_ReceiveOrderReference -->
  <testcase classname="automationassignment.TestCase2Bonus" name="step4_CaptureDetailsForDelivery" time="0.000">
    <skipped/>
  </testcase> <!-- step4_CaptureDetailsForDelivery -->
</testsuite> <!-- automationassignment.TestCase2Bonus -->
